src/components/button/button.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import './button.css'
+
+
+export default function Button(props: any): React.ReactElement {
+    return <button
+        className={props.className}
+        onClick={() => props.onClick()}
```

Comment:
"Using an arrow function in render creates a new function each time the component renders, which may break optimizations based on strict identity comparison." try `onClick={props.onClick}`

Reply:
fixed